derp/derp_server.go

```diff
@@ -391,6 +391,18 @@ func (s *Server) isClosed() bool {
 	return s.closed
 }
 
+// IsClientConnected reports whether the client with specified key is connected.
+// This is used in tests to verify that nodes are connected to derp.
```

Comment:
to DERP? or just "are connected." 

Are you going to poll this? 😬  I guess if it deflakes.

Can you rename to make it include `ForTest` in the name? See `git grep ForTest` in the root for examples.

Reply:
Yeah I'm polling this, would be nice to have a callback or something. But that can happen in the future.
Renamed to `IsClientConnectedForTest`